demo/demo.cpp

```diff
@@ -14,13 +14,11 @@
 
 #include "model_loader_assimp.hpp"
 
-#define SCENE viknell_scene
+#define SCENE emibl_scene
```

Comment:
Lets use viknell as default for now. Its better for testing rtx features.

Reply:
👍